apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss

```diff
@@ -198,6 +198,31 @@ p {
     }
   }
 
+  .section {
+    @apply com-px-6 com-my-12 com-mx-72px;
+
+    div {
+      @apply com-flex com-flex-row com-justify-between com-m-0 com-p-0 com-gap-x-72;
+
+      .content {
+        @apply com-mt-12 com-mx-4 com-flex com-flex-col;
+
+        h1 {
+          @apply com-mb-4 com-p-0 com-text-4xl com-font-extrabold com-mt-0;
```

Comment:
h1 already has a 36px font size, then why we are giving font size here?

Reply:
Solved